### ISSUE TITLE
Remove aria warning for Image Viewer Basic component

### ISSUE THREAD

**Issue by marcello-stackzero** [stackzero-labs/ui#12]
**Describe the bug**

image-viewer-basic - Warning: Missing `Description` or `aria-describedby={undefined}` for {DialogContent}.

**To Reproduce**
1. Go to image-viewer-basic
2. Open an image
3. See warning

**Expected behavior**
There should be no warning.
